cfc-testing/comprehensive-resource-validator.sh

```
@@ -226,11 +249,37 @@ synthesize_and_validate() {
+        context_flags="$context_flags --context cfc.minInstanceCapacity=1"
+        context_flags="$context_flags --context cfc.maxInstanceCapacity=3"
+        context_flags="$context_flags --context cfc.cpuTargetUtilization=60"
+    fi
+
+    if [[ "$domain_config" == "with-domain" ]]; then
+        context_flags="$context_flags --context cfc.domain=$DOMAIN"
+        context_flags="$context_flags --context cfc.createZone=true"
+        if [[ "$subdomain_config" == "with-subdomain" ]]; then
+            # Use a short subdomain to avoid ACM 64-character limit
+            # Create a hash of the stack name to keep it unique but short
+            local subdomain="test-$(echo $stack_name | md5 | cut -c1-8)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)